Add fmu exported by Altair-Activate

masoud-najafi opened the pull request:
I have added a few FMUs  exported by Altair-Activate for Win64, Linux65 for Both ME and CS types. These FMUs successfully passed the FMUChecker tests.

t-sommer reviewed (changes requested):
Your FMUs fail to validate. Please fix your FMUs and run `validate_repo.py` before opening a PR.

masoud-najafi commented:
Hi Torsten,
I had tested my FMUs before requesting the PR and the FMUs had already passed the test with FMPY 0.2.9.
But, in doubt, I took a fresh FMPY 0.2.26 and a fresh CrossCheck from GIT and did the test again. There is this error "already present" in the git base (this error is not from my FMUs).
#################################
1 problems found in E:\FMI\cross_check_git3\fmi-cross-check
Validated 1464 FMUs and 13525 results
#################################
E:\FMI\cross_check_git3\fmi-cross-check\fmus\2.0\cs\linux64\pandapower\2.1.0\european_lv\european_lv_in.csv is larger than 1 MB (1.00155 MB)

I fixed this error locally, and added my FMUs and the tests went fine, no problem.

#################################
0 problems found in E:\FMI\cross_check_gi2\fmi-cross-check
Validated 1488 FMUs and 13525 results
#################################


Maybe this is the cause of the failure.

Best,
Masoud


De : Torsten Sommer <notifications@github.com>
Envoyé : lundi 1 février 2021 23:32
À : modelica/fmi-cross-check <fmi-cross-check@noreply.github.com>
Cc : Masoud Najafi <masoud@altair.com>; Author <author@noreply.github.com>
Objet : Re: [modelica/fmi-cross-check] Add fmu exported by Altair-Activate (#101)


@t-sommer requested changes on this pull request.

Your FMUs fail to validate. Please fix your FMUs and run validate_repo.py before opening a PR.

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmodelica%2Ffmi-cross-check%2Fpull%2F101%23pullrequestreview-580825848&data=04%7C01%7Cmasoud%40altair.com%7C7ebbd9315557429294dc08d8c70130be%7C2bae5b570eb848fbba47990259da89d2%7C0%7C0%7C637478155220787147%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=b6UaO%2B3be10%2BVWvkBWzwgq4iFZRUA6XzLG%2FumM073qI%3D&reserved=0>, or unsubscribe<https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAH7O42L6L3RWVTD6X3JPRT3S44TV7ANCNFSM4T46B3BA&data=04%7C01%7Cmasoud%40altair.com%7C7ebbd9315557429294dc08d8c70130be%7C2bae5b570eb848fbba47990259da89d2%7C0%7C0%7C637478155220787147%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=HE%2FsvKyO3NV59HHJJGzVns2R%2BakRW5YLLyqBr%2BgLjs4%3D&reserved=0>.
